tests/test_bgp_neighbor_af.rb

```diff
@@ -117,6 +184,15 @@ def test_nbrs_with_masks
     @@matrix.each do |k, v|
       asn, vrf, nbr, af = v
       nbr += (nbr[/:/]) ? '/64' : '/16'
+      if platform == :ios_xr
+        # XR does not do slash masks and neighbors
+        # so make sure to catch the error
+        assert_raises(ArgumentError, 'Neighbors do not support /') do
```

Comment:
Yeah fixed. The raise argumenterror was working because it was catching a different argument error.